public/app/plugins/datasource/grafana-testdata-datasource/plugin.json

```diff
@@ -38,6 +38,6 @@
   ],
 
   "dependencies": {
-    "grafanaDependency": ">=5.0.0"
+    "grafanaDependency": ">=10.3.0"
```

Comment:
Hmm not sure this includes Grafana 10.0.3-pre releases? Gcom uses https://semver.npmjs.com/ for semver comparison. I have some vague memory something additional is needed so feel free to check

Reply:
It should: https://jubianchi.github.io/semver-check/#/%3E%3D10.3.0/10.3.0-63036

Reply:
Okay great

Reply:
Hmm I think it's the go library that we use that has some oddness around semver comparision and prerelease versions... https://pkg.go.dev/github.com/Masterminds/semver/v3#readme-working-with-prerelease-versions

Reply:
it seems that `gen-cue` doesn't like pre releases :roll_eyes: